src/Stratis.Bitcoin.Apps.Browser/Interfaces/IAppsService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Stratis.Bitcoin.Apps.Browser.Dto;
+
+namespace Stratis.Bitcoin.Apps.Browser.Interfaces
+{
+    public interface IAppsService
```

Comment:
Hi Paul please add comments on this interface to describe the intention

Reply:
OK